### ISSUE TITLE
Add link to people form somewhere on the site

### ISSUE THREAD

**Issue by simonv3** [opensourcedesign/opensourcedesign.github.io#147]
Right now the [people form](http://opensourcedesign.net/people-form/) exists, but it's not findable. We should let people find it. Where does it make sense to add?

I think overall it would make sense in a "Are you a designer" section, but we'd need one like that.

https://github.com/opensourcedesign/organization/pull/100

cc @evalica

**Comment by bnvk** [comment]
I think this should be a "call to action" as part of something like **Join Us** or **Participate** which also plays into the [contribute page](https://opensourcedesign.net/contribute/) which I also think needs some love and to be more featured. I have some ideas, but those of others are much appreciated :)

**Comment by Erioldoesdesign** [comment]
I agree and think this is more of a question around how to rework our contribute page or create a join us page. It's also a little bit connected to the 'showcase' on the landing page: https://github.com/opensourcedesign/opensourcedesign.github.io/issues/291

I also have a couple of issues with the people form in general...I think it's available or booked which would need some way for the people who have listed themselves to change it as and when they become available...that edges into logins/my account territory. 
Also I think that information from the form is missing on any individual profile pages? for example: https://opensourcedesign.net/people/#belen-pena doesn't offer any more of that detail from the people form.

I think I would prefer a list of people's names and a simple link to their website or portfolios? I think maintaining a list of changing information of designers is a bit ambitious! In fact I think the simple instructions on: https://opensourcedesign.net/people/ about 'add yourself' is a good simple option!

**Comment by Erioldoesdesign** [comment]
Though I do think it'd be simple and relevant to include 'Check out other [open source designers](https://opensourcedesign.net/people/) and add yourself to the list!' somewhere in the right column of text here:

<img width="1159" alt="Screenshot 2021-06-22 at 20 45 41" src="https://user-images.githubusercontent.com/11681324/122989770-1c1d3580-d39b-11eb-9361-2aa6e3d71509.png">

**Comment by sharmavasant** [comment]
add a "JOIN US" page

**Comment by jdittrich** [comment]
Do we still want to have a list? If no strong "yes" (also given that is needs maintainance), I suggest closing.
